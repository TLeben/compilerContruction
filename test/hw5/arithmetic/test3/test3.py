#!/usr/bin/env python

def change(list):
    list[0] = 99
    return

l = [4,6,8]
print l
change(l)
print l