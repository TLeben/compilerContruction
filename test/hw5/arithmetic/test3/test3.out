[4, 6, 8]
[99, 6, 8]
